src/main/java/org/mule/module/json/internal/cleanup/JsonModuleResourceReleaser.java

```diff
@@ -54,28 +64,18 @@ public JsonModuleResourceReleaser(SchedulerService schedulerService) {
   public synchronized void releaseExecutors() {
     LOGGER.debug("Stopping the known executors services");
     Field bundleField = null;
-    boolean isAccessible = false;
     try {
       bundleField = JsonNodeReader.class.getDeclaredField("BUNDLE");
```

Comment:
I noticed BUNDLE is repeated four times, though not in your recent changes. Consider using a constant for maintainability. Thoughts?

Reply:
Sure, we can do that.